src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -1421,6 +1421,30 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
         current_scope->add_symbol(var_name, ASR::down_cast<ASR::symbol_t>(v));
     }
 
+    void add_lpython_version(const Location &loc) {
+        std::string var_name = "__LPYTHON_VERSION__";
+        std::string var_value = "0.2.0-1041-g32c1736d9";
```

Comment:
```suggestion
        std::string var_value = FORTRAN_VERSION;
```
and include `<libasr/config.h>`. That should work.